modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/infrastructure/ApiClient.kt.mustache

```diff
@@ -25,16 +26,32 @@ import {{packageName}}.auth.*
 {{#nonPublicApi}}internal {{/nonPublicApi}}open class ApiClient(
         private val baseUrl: String,
         httpClientEngine: HttpClientEngine?,
+        httpClientConfig: ((HttpClientConfig<*>) -> Unit)?,
```

Comment:
@tkirshboim can you please make this with null value by default and remove the extra constructor?
```
httpClientConfig: ((HttpClientConfig<*>) -> Unit)? = null,
```

Reply:
Thanks, let's wait for the CI to finish 👍

Reply:
@4bruno CI passed. ☺️